### PR TITLE
Use ingress controller instead of classic ELBs in all project templates

### PR DESCRIPTION
**Pull request by joeyAghion**
I didn't validate these locally, but tried to carefully copy+paste from projects that I know work.

### CODE REVIEW

File: default/hokusai/production.yml.j2 [hunk 1]
```diff
@@ -129,28 +98,33 @@ metadata:
...
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {% raw %}{{ project_name }}{% endraw %}
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ cloudflareIpSourceRanges|join(',') }}"
+spec:
+  rules:
+    - host: {% raw %}{{ project_name }}{% endraw %}.artsy.net
```
Comment (joeyAghion) [inline default/hokusai/production.yml.j2:124]: Note that we shouldn't assume projects' code names are used as their URLs. (More often than not, the URLs are different.) But it seems like a decent starting point.

File: nodejs/hokusai/production.yml.j2 [hunk 2]
```diff
@@ -34,7 +34,7 @@ spec:
               value: "8080"
             - name: NODE_ENV
               value: production
-            - name: TRACE_AGENT_HOSTNAME
+            - name: DATADOG_TRACE_AGENT_HOSTNAME
```
Comment (joeyAghion) [inline nodejs/hokusai/production.yml.j2:37]: This is an unrelated change but I _think_ closer to our common practice.

File: default/hokusai/production.yml.j2 (hunk 1 above)
Comment (artsyjian) [reply default/hokusai/production.yml.j2:124]: agreed. should be pretty obvious to someone that it must be changed.

File: nodejs/hokusai/production.yml.j2 (hunk 2 above)
Comment (artsyjian) [reply nodejs/hokusai/production.yml.j2:37]: sure.

**Comment by izakp** [comment]
👏 nice one

**Pull request merged**